server/views/pages/overview/partials/timelineTab/timelineTabContents.njk

```diff
@@ -2,57 +2,26 @@
-
-        {% for event in timeline.events %}
-          {% if event.eventType == 'PRISON_ADMISSION' or event.eventType == 'PRISON_TRANSFER' or event.eventType == 'PRISON_RELEASE' %}
-            <div class="moj-timeline__item" data-qa-event-type="{{ event.eventType }}">
-              <div class="moj-timeline__header">
-                <h2 class="moj-timeline__title">{{ event | formatPrisonMovementEvent }}</h2>
-              </div>
-
-              <p class="moj-timeline__date">
-                <time datetime="{{ event.timestamp }}">{{ event.timestamp | formatDate('D MMMM YYYY') }}</time>
-              </p>
-
-          {% else %}
-            <div class="moj-timeline__item" data-qa-event-type="{{ event.eventType }}">
-              <div class="moj-timeline__header">
-                <h2 class="moj-timeline__title">{{ event.eventType | formatTimelineEvent }}</h2>
-                <p class="moj-timeline__byline govuk-!-display-none-print">by {{ event.actionedByDisplayName }}, {{ event.prison.prisonName if event.prison.prisonName else event.prison.prisonId }}</p>
-              </div>
-
-              <p class="moj-timeline__date">
-                <time datetime="{{ event.timestamp }}">{{ event.timestamp | formatDate('D MMMM YYYY') }}</time>
-              </p>
-
-              {% if event.eventType == 'ACTION_PLAN_CREATED' or event.eventType == 'GOAL_UPDATED' or event.eventType == 'GOAL_CREATED' or event.eventType == 'MULTIPLE_GOALS_CREATED' %}
-              <div class="moj-timeline__description govuk-!-display-none-print">
-                <p class="govuk-body govuk-!-margin-bottom-0">
-                  <a href="/plan/{{ prisonerSummary.prisonNumber }}/view/overview" data-id="timeline-prisoner-{{ prisonerSummary.prisonNumber }}">
-                    {% if event.eventType == 'ACTION_PLAN_CREATED' %}
-                      View {{ prisonerSummary.firstName }} {{ prisonerSummary.lastName | title }}'s learning and work progress
-                    {% elif event.eventType == 'GOAL_UPDATED' or event.eventType == 'GOAL_CREATED' or event.eventType == 'MULTIPLE_GOALS_CREATED' %}
-                      View <span class="govuk-visually-hidden">{{ prisonerSummary.firstName }} {{ prisonerSummary.lastName | title }}'s </span>goals
-                    {% endif %}
-                  </a>
-                </p>
-              </div>
-              {% endif %}
-          {% endif %}
-          </div>
-        {% endfor %}
-
+          {% set supportedTimelineEventTypes = ['ACTION_PLAN_CREATED', 'INDUCTION_UPDATED', 'GOAL_UPDATED', 'GOAL_CREATED', 'MULTIPLE_GOALS_CREATED', 'PRISON_ADMISSION', 'PRISON_RELEASE', 'PRISON_TRANSFER'] %}
+
+          {% for event in timeline.events %}
+            {% if event.eventType in supportedTimelineEventTypes %}
+              {% include './_timelineEvent-' + event.eventType + '.njk' %}
+            {% else %}
+              {# A timeline event type we dont currently support in the view. Do nothing and skip it #}
```

Comment:
I've added this `else` block for no reason other to delineate where it is should we ever get asked to implement some kind of "catch all" or similar (I doubt we ever would - happy to remove the else block and comment if we think it's nothing but noise)

Reply:
I think it's self-documenting already without the else so let's remove it.

Reply:
Done 👍